### PR TITLE
[f] Contain/exclude multiple args

### PR DESCRIPTION
**Pull request by catriuspham**
### Related issue
https://github.com/EastAgile/robber.py/issues/11

### Description
Now contain/exclude can take multiple args

```python
expect(a).to.contain(x, y, z)
expect(a).to.exclude(x, y, z)
```

### CODE REVIEW

File: robber/matchers/contain.py [hunk 1]
```diff
@@ -5,16 +5,33 @@
...
+        if not self.is_negative:
+            for expected in expected_list:
+                if expected not in self.actual:
+                    self.expected_arg = expected
+                    return False
+
+            return True
+        else:
+            # As this is the negative case, we have to flip the return value.
+            for expected in expected_list:
```
Comment (hieueastagile) [inline robber/matchers/contain.py]: I guess we can put this loop into a function and return its negative form, the result is still the same, right?
```python
def is_contained(expected, expecteds):
  ..

contained = is_contained(expected, expecteds)
return contained if self.is_negative else (not contained) # not sure if syntax is correct here.
```

Comment (catriuspham) [reply robber/matchers/contain.py]: A contains B == A contains all elements of B (1) (B1 in A and B2 in A and ... Bn in A)
A not contains B == A not contains all elements of B (2) (B1 not in A and B2 not in A and ... Bn not in A)
not (1) != (2)

I cannot find another better way to explain, hope you understand :sweat_smile:

**Comment by catriuspham** [comment]
@hieueastagile This PR is rather old, please take a look and see if we can merge it, thanks :smile:

File: robber/matchers/contain.py (hunk 1 above)
Comment (hieueastagile) [reply robber/matchers/contain.py]: Sorry for taking this too long @catriuspham. I got your idea, but I guess we can refactor it a bit for easier to read with `any`, `all`.
```python
contained = all([b in A for b in B])
not_contained == all([b not in A for b in B]) == any([b in A for b in B])
```
And I guess it's better to split it to 2 functions.

Comment (hieueastagile) [reply robber/matchers/contain.py]: @catriuspham: See this already?

Comment (catriuspham) [reply robber/matchers/contain.py]: @hieueastagile Ah yes, I've finished another PR, gonna look into this soon.

Comment (catriuspham) [reply robber/matchers/contain.py]: @hieueastagile I have to get the included/excluded elements so `all` and `any` just don't work. I end up using set's `difference` and `intersection`. It's easy to read too.

**Comment by catriuspham** [comment]
@hieueastagile Did you see the update I made some days ago?

**Comment by hieueastagile** [comment]
I saw it, but I think do you think that we still can improve the readability here? I know it's annoying and I'm sorry about that but I still see it could be improved more :(.

**Comment by catriuspham** [comment]
@hieueastagile Best I can do is to create the function `_get_first` to DRY it a bit. Please check if you think it's better now :smile:

**Comment by hieueastagile** [comment]
LGTM, let's merge this, thanks @catriuspham

**Comment by catriuspham** [comment]
@oyster Please help merge this and close the following issue: https://github.com/EastAgile/robber.py/issues/11 thanks.

**Comment by oyster** [comment]
All requested pull requests and relevant issues should be merged and closed now. Please let me know if you need anything else, @catriuspham.

**Pull request merged**